### ISSUE TITLE
Set branch 'develop' as default

### ISSUE THREAD

**Issue by albertinisg** [Bitergia/fiware-devguide-app#67]
One week ago we agreed to start using `develop` branch instead of `master`.  After that, PR's #55 (which was already accepted) and #65 went against `master` branch, so we will need now to sync the branches.

@jmcanterafonseca I don't have the permission to set the default branch (described [here](https://help.github.com/articles/setting-the-default-branch/)). Could you please update it?

@FiwareULPGC please, take note of this change. All the new developments must go against `develop`, and `master` will be the branch used for releasing versions.

**Comment by dmabtrg** [comment]
Also, I think we should rebase `develop` on top of current `master` before merging any of the open pull requests.

**Comment by dmabtrg** [comment]
Following the change from `master` to `develop` as default branch, the `develop` branch was some commits behind `master`.  To fix this before more changes land on the branch, we've just rebased the `develop` branch on top of `master`, making the `develop` branch up to date with the latest changes that were merged to `master`.  This change requires you to update your local copy of the repository and your forks to sincronize the branch.  To do this, follow this steps:
- First, make a backup copy of your local repository, just in case something goes horribly wrong.  If using linux, this should be enough:
  
  ```
  cp -a my-local-repository my-local-repository-$( date +%Y%m%d )
  ```
- In case you have a local copy of the main repository:
  
  ```
  git checkout develop
  git pull --rebase
  ```
- For your local copy of the fork repository:
  - Get the latest changes from the `upstream` repository:
    
    ```
    git fetch upstream
    ```
  - If you have local uncommited changes, stash them before rebasing
    
    ```
    git stash
    ```
  - Rebase the `develop` branch:
    
    ```
    git rebase upstream/develop develop
    ```
  - Rebase any `branch` stemmed from `develop
    `
    
    ```
    git rebase develop <branch>
    ```
  - Push changes to your fork on github
    
    ```
    git push --force origin develop
    git push --force origin <branch>
    ```

**Remember that all development must be done against the `develop` branch.**
